pkg/component/nodemanagement/dependencywatchdog/access_test.go

```diff
@@ -21,7 +21,7 @@ import (
 	. "github.com/onsi/gomega"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/utils/pointer"
+	"k8s.io/utils/ptr"
```

Comment:
As you introduced a change in that direction. There is 1 more usage of `k8s.io/utils/pointer` that was introduced 21h ago: https://github.com/gardener/gardener/blob/d8561d9addbf1786c90d853cfe6df2f05329f249/pkg/gardenlet/operation/botanist/kubeapiserverexposure_test.go#L114. You adapt it as well.